### PR TITLE
[Settings]: Create setting to show service and shared workers in the target list (will now hide by default)

### LINKED ISSUE #276: Filter extensions and service workers from list of targets
In the list of targets, I can connect to many targets, even though I only have one tab open in Edge. I'm not sure if there is a use  case for connecting to chrome-extensions and service workers. If not, I think it would be a good idea to limit the list of targets to actual tabs.

### PR DESCRIPTION
**Pull request by mliao95**
This PR hides service and shared workers behind a settings flag to declutter the target list.

GIF:
settings-show-workers

closes #276

### CODE REVIEW

File: src/cdpTargetsProvider.ts [hunk 1]
```diff
@@ -43,9 +43,7 @@ export default class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTa
                         let targetsProcessed = 0;
                         responseArray.forEach(async (target: IRemoteTargetJson) => {
                             const actualTarget = fixRemoteWebSocket(hostname, port, target);
-                            if (actualTarget.type !== 'page') {
-                                targets.push(new CDPTarget(actualTarget, "", this.extensionPath));
-                            } else {
+                            if (actualTarget.type !== 'service_worker' && actualTarget.type !== 'shared_worker') {
```
Comment (vidorteg) [inline src/cdpTargetsProvider.ts:46]: Is there a scenario where we want to use the extension for service worker debugging? Maybe with the network tab? If this is the case I think a setting like like:
"Show service workers on target list" and make it false by default makes more sense. Thoughts?

**Review by vidorteg — CHANGES_REQUESTED** [review]
Left a comment

File: src/cdpTargetsProvider.ts (hunk 1 above)
Comment (mliao95) [reply src/cdpTargetsProvider.ts:46]: I like that idea just so we don't break any existing workflows.  Will work on a fix now.

Comment (mliao95) [reply src/cdpTargetsProvider.ts]: Fixed.  See new GIF in description

**Review by vidorteg — APPROVED** [review]
Nice :)

**Pull request merged**